### ISSUE TITLE
[Bug]: Apps do not compile with flutter 3.29.0

### ISSUE THREAD

**Issue by timobaehr** [ravitejaavv/store_checker#36]
## Steps to reproduce

Create a project and use store_checker: ^1.6.0 using flutter 3.29.0.

It shows build error:
```
Note: /Users/timo/.pub-cache/hosted/pub.dev/in_app_review-2.0.10/android/src/main/java/dev/britannio/in_app_review/InAppReviewPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
/Users/timo/.pub-cache/hosted/pub.dev/store_checker-1.6.0/android/src/main/java/store/checker/store_checker/StoreCheckerPlugin.java:14: error: cannot find symbol
import io.flutter.plugin.common.PluginRegistry.Registrar;
                                              ^
  symbol:   class Registrar
  location: interface PluginRegistry
/Users/timo/.pub-cache/hosted/pub.dev/store_checker-1.6.0/android/src/main/java/store/checker/store_checker/StoreCheckerPlugin.java:30: error: cannot find symbol
  public static void registerWith(Registrar registrar) {
                                  ^
  symbol:   class Registrar
  location: class StoreCheckerPlugin
Note: /Users/timo/.pub-cache/hosted/pub.dev/store_checker-1.6.0/android/src/main/java/store/checker/store_checker/StoreCheckerPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
2 errors

FAILURE: Build failed with an exception.
```

## Expected results

Do not show an error

## Actual results

Show an error in flutter 3.29.0

## Version

1.6.0

## Related issues

https://github.com/Baseflow/flutter-google-api-availability/issues/51

**Comment by husainazkas** [comment]
Merge PR #37 will fix this issue. You can keep waiting or use contributor's own branch

**Comment by Charlinjoeaht** [comment]
When are you planning to fix this in production? We are blocked

**Comment by ravitejaavv** [comment]
@Charlinjoeaht request you to upgrade from 1.6.0 to 1.7.0  

Please open this thread for any issues with the latest version
